modules/lambda/variables.tf

```diff
@@ -268,7 +269,37 @@ variable "iam_policy_description" {
 
 variable "policy_json" {
   type        = string
-  description = "IAM policy to attach to the Lambda IAM role"
+  description = "IAM policy to attach to the Lambda IAM role. This can be used with or instead of the `var.iam_policy`."
```

Comment:
```suggestion
  description = "IAM policy to attach to the Lambda role, specified as JSON. This can be used with or instead of `var.iam_policy`."
```